readthedocs/docsitalia/serializers.py

```diff
@@ -3,14 +3,37 @@
 from __future__ import absolute_import
 
 from rest_framework import serializers
+from readthedocs.restapi.serializers import ProjectSerializer
 
-from readthedocs.projects.models import Project
 
-
-class ProjectsByTagSerializer(serializers.HyperlinkedModelSerializer):
+class ProjectsByTagSerializer(ProjectSerializer):
 
     """Projects by Tag DRF Serializer"""
 
-    class Meta:
-        model = Project
-        fields = ('name', 'description', 'url')
+    publisher = serializers.SerializerMethodField()
+    publisher_project = serializers.SerializerMethodField()
+
+    class Meta(ProjectSerializer.Meta):
+        fields = (
+            'id', 'name', 'slug', 'description',
+            'canonical_url', 'publisher', 'publisher_project',
+        )
+
+    @classmethod
```

Comment:
c'è un motivo del perchè questi siano classmethod?

Reply:
per pylint

Reply:
meglio uno staticmethod?

Reply:
obey the pylint god!